public/app/features/users/__snapshots__/UsersListPage.test.tsx.snap

```diff
@@ -20,11 +20,25 @@ exports[`Render should render List page 1`] = `
       onShowInvites={[Function]}
```

Comment:
I think we can get rid of this snapshot rather than updating it

Reply:
Sorry @dprokop I merged before refresh which meant I missed this suggestion. Will follow up with a PR that moves the tests to RTL.

Reply:
no worries!